rpm/imagemagick.spec

```diff
@@ -1,23 +1,28 @@
-%global VER 6.9.12
+%global VER 6.9.13
 
 Name:		ImageMagick
-Version:	6.9.12.12
+Version:	6.9.13.11
 Release:	1%{?dist}
 Summary:	An X application for displaying and manipulating images
```

Comment:
Old, but "X", eh :)

Reply:
Yeah, it seems Fedora spec still had that also, removed the X from it now.